### PR TITLE
Kyma-integration k8s 1.14 image

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:

- This is the first of images used in kyma integration pipelines (with kubectl 1.14)

**Related issue(s)**

### CODE REVIEW

File: prow/images/kyma-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Kyma-integration images
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
# Kyma integration images
```

File: prow/images/kyma-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Overview
+
+This folder contains the image with tools necessary for provisioning kyma-integrtion clusters.
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
This folder contains the image with tools that are necessary to provision Kyma integration clusters.
```

File: prow/images/kyma-integration/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The image consists of:
+- Go
+- Dep
+- Helm
+- Gcloud
+- Az
+- Docker
+
+# Maintaning the image
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
## Maintaining the image
```

File: prow/images/kyma-integration/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Google Cloud SDK comes with one default `kubectl` version and couple additional ones.
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
Google Cloud SDK comes with one default `kubectl` version and a couple of additional ones.
```

File: prow/images/kyma-integration/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+See details [here](https://cloud.google.com/sdk/docs/release-notes#27600_2020-01-14).
+
+`CLUSTER_VERSION` variable is used to match `kubectl` version used in the image with the cluster version that is build by ProwJob pipeline.
+
+```
+ENV CLUSTER_VERSION=1.14
+RUN mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
+```
+
+**CAUTION** Each image is tagged with corresponding `kubectl` version. While updating the image please do not forget to adjust the tag.
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
>**CAUTION:** Each image is tagged with the corresponding `kubectl` version. When updating the image, do not forget to adjust the tag. 
```

File: prow/images/kyma-integration/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- Dep
+- Helm
+- Gcloud
+- Az
+- Docker
+
+# Maintaning the image
+
+Google Cloud SDK comes with one default `kubectl` version and couple additional ones.
+See details [here](https://cloud.google.com/sdk/docs/release-notes#27600_2020-01-14).
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
See the details [here](https://cloud.google.com/sdk/docs/release-notes#27600_2020-01-14). To set the version of `kubectl` other than the default, run:
```

Comment (klaudiagrz) [reply prow/images/kyma-integration/README.md]: Please move this command here:
```
ENV CLUSTER_VERSION=1.14
RUN mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
```

File: prow/images/kyma-integration/README.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+`CLUSTER_VERSION` variable is used to match `kubectl` version used in the image with the cluster version that is build by ProwJob pipeline.
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
- `CLUSTER_VERSION` variable matches the `kubectl` version used in the image with the cluster version that is built by the ProwJob pipeline.
```

File: prow/images/kyma-integration/README.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+ENV CLUSTER_VERSION=1.14
+RUN mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
+```
+
+**CAUTION** Each image is tagged with corresponding `kubectl` version. While updating the image please do not forget to adjust the tag. 
+```
+docker tag $(IMG_NAME) $(IMG):k8s-1.14
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
docker tag ${IMG_NAME} ${IMG}:k8s-1.14
```

File: prow/images/kyma-integration/README.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+# Maintaning the image
+
+Google Cloud SDK comes with one default `kubectl` version and couple additional ones.
+See details [here](https://cloud.google.com/sdk/docs/release-notes#27600_2020-01-14).
+
+`CLUSTER_VERSION` variable is used to match `kubectl` version used in the image with the cluster version that is build by ProwJob pipeline.
+
+```
+ENV CLUSTER_VERSION=1.14
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]:
```suggestion
ENV CLUSTER_VERSION={VERSION}
```

File: prow/images/kyma-integration/README.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+RUN mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
+```
+
+**CAUTION** Each image is tagged with corresponding `kubectl` version. While updating the image please do not forget to adjust the tag. 
+```
```
Comment (klaudiagrz) [inline prow/images/kyma-integration/README.md]: Add such sentence before the command:
```To adjust the tag on the updated Docker image, run:```

Comment (kasiakepka) [reply prow/images/kyma-integration/README.md]: It is not a command that you run separately. This line comes from Makefile and the only thing you have to do is to update the `k8s-1.14` tag.

File: prow/images/kyma-integration/README.md (hunk 8 above)
Comment (kasiakepka) [reply prow/images/kyma-integration/README.md]: This is "copy+paste" from Makefile so I think we should not change it in README.md.

**Pull request merged**